.github/workflows/build.yml

```diff
@@ -88,7 +88,7 @@ jobs:
     - uses: actions/checkout@v4
       with: {fetch-depth: 0, submodules: recursive}
     - name: set requirements
-      run: sed -ri -e '/ python=/d' -e 's/(.* numpy=).*/\1${{ matrix.numpy-version }}/' -e 's/=cuda*//' -e '/tigre/d' scripts/requirements-test.yml
+      run: sed -ri -e '/ python=/d' -e 's/(.* numpy=).*/\1${{ matrix.numpy-version }}/' scripts/requirements-test.yml    
```

Comment:
Should revert this change, it's pointlessly installing CUDA on a CPU system

Reply:
With TIGRE not installed sphinx doesn't render the documentation for the TIGRE plugin. Do you see other ways to render the documentation?

Reply:
this line isn't part of the docs job.

Reply:
```suggestion
      run: sed -ri -e '/ python=/d' -e 's/(.* numpy=).*/\1${{ matrix.numpy-version }}/' -e 's/=cuda*//' -e '/tigre/d' scripts/requirements-test.yml
```